salt/flink/init.sls

```diff
@@ -27,6 +27,17 @@
 {% set hadoop_home_bin = '/opt/cloudera/parcels/CDH/bin' %}
 {% endif %}
 
+# edge node IP
+{% set edge_nodes = salt['pnda.get_hosts_for_role']('hadoop_edge') %}
```

Comment:
Use the specific graphite role, rather than assuming it will be on the hadoop-edge node.